react-search-component/my-app/src/SearchBar.jsx

```diff
@@ -0,0 +1,13 @@
+export default function SearchBar({ handleSearch }) {
+  return (
+    <div>
+      <input
+        type="text"
+        name="searchForm"
+        placeholder="🔍Search your word"
```

Comment:
This isn't quite what was expected, though I'll not worry about it for this review. With this approach, the magnifying glass disappears as soon as the user starts typing. The intent was to have the magnifying glass always visible. You can do that by rendering the magnifying glass separately and using CSS to place it where you want and to indent the input text.